package.json

```diff
@@ -13,6 +13,7 @@
     "connect-livereload": "~0.5.0",
     "connect-modrewrite": "~0.7.9",
     "coveralls": "~2.11.2",
+    "grunt-bump": "^0.1.0",
     "grunt-contrib-clean": "~0.6.0",
```

Comment:
you should use `~` instead of `^` to avoid grunt updating the package to a major release which could have breaking functionality